extensions/vault/runtime/src/main/java/io/quarkus/vault/internal/VaultAdManager.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.vault.internal;
+
+import org.jboss.logging.Logger;
+
+import io.quarkus.vault.internal.client.VaultClient;
+
+public class VaultAdManager {
+
+    private static final Logger log = Logger.getLogger(VaultAdManager.class.getName());
+
+    private VaultAuthManager vaultAuthManager;
+    private VaultClient vaultClient;
+
+    public VaultAdManager(VaultAuthManager vaultAuthManager, VaultClient vaultClient) {
+        this.vaultAuthManager = vaultAuthManager;
+        this.vaultClient = vaultClient;
+    }
+
+    public String getCreds(String roleName) {
```

Comment:
Let's use `Credentials` instead? Better be consistent.

Reply:
this class is gone. see https://github.com/quarkusio/quarkus/pull/2897#discussion_r326852529